src/main/index.ts

```diff
@@ -93,8 +93,11 @@ if (!app.requestSingleInstanceLock()) {
 
   for (const arg of process.argv) {
     if (arg.toLowerCase().startsWith("lens://")) {
-      lprm.route(arg)
-        .catch(error => logger.error(`${LensProtocolRouterMain.LoggingPrefix}: an error occured`, { error, rawUrl: arg }));
+      try {
```

Comment:
Should we refactor this logic to a function? I'm seeing exactly the same code on lines 108-116.

Reply:
Sure, I'll do that in a follow up PR.